lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -84,6 +86,13 @@ class CourseEnrollmentSerializer(serializers.ModelSerializer):
     """
     course = CourseOverviewField(source="course_overview", read_only=True)
     certificate = serializers.SerializerMethodField()
+    expiration = serializers.SerializerMethodField()
```

Comment:
Note in the spec, I had put in "expires" instead of "expiration", Marco suggested that might be unclear as well

Reply:
changed to `audit_access_expires`